projects/canopy/src/lib/button/button.stories.ts

```diff
@@ -27,25 +28,29 @@ const contentGroupId = 'content';
   template: `
     <button
       lg-button
-      [variant]="variant"
+      [disabled]="disabled"
+      [fullWidth]="fullWidth"
       [iconButton]="iconButton"
       [iconPosition]="iconPosition"
       [loading]="loading"
-      [fullWidth]="fullWidth"
+      [size]="size"
```

Comment:
I'm wondering if maybe we should have a single `size` input that includes `fullWidth` instead of having them as two separate inputs. They both affect the size of the button so I think it makes sense 🤔

Reply:
I guess the downside of doing this is that you may want a button to be both `sm` _and_ `fullWidth`, which would be easy to do with separate inputs. If they were combined, we may need more options such as `sm-full-width`, `md-full-width` and so on, which could get long.

Reply:
Yeah it might a good idea, it could be linked to this issue.. https://github.com/canopy-collective/canopy/issues/251. I didn't actually change anything around that on this PR, I just put it into alphabetical order. (which I realise doesn't make it easier for the reviewer :-) )